app/components/VideoControls.js

```
@@ -0,0 +1,70 @@
+  onToggleMic, 
+  onToggleScreenShare,
+  onCopyRoomLink,
+  onLeaveRoom
+}) {
+  return (
+    <div className="flex flex-wrap justify-center gap-2 mt-4">
+      <button 
+        className={`${isMicOn ? 'bg-blue-500 hover:bg-blue-600' : 'bg-red-500 hover:bg-red-600'} text-white px-4 py-2 rounded transition-colors flex items-center gap-2`}
+        onClick={onToggleMic}
+        title={isMicOn ? "Turn off microphone" : "Turn on microphone"}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)